bin/generate_fake_data.rb

```diff
@@ -0,0 +1,111 @@
+#!env ruby
+
+require "factory_bot"
+require "faker"
+require_relative "../spec/fixtures/organizations"
+require_relative "../spec/fixtures/collections"
+
+FactoryBot.find_definitions
+
+Services.register(:ht_organizations) { mock_organizations }
+Services.register(:ht_collections) { mock_collections }
+
+OUTDIR = "/usr/src/app/testdata"
+MAX_OCN = 100000
+
+def random_year
+  Faker::Date.between(from: Date.parse("1850-01-01"), to: Date.today).year
+end
+
+def warn_exists(outfile)
+  if(File.exist?(outfile))
+    puts "#{outfile} already exists, not creating"
+    true
+  else
+    false
+  end
+end
+
+
+# For now, only single-part monographs
+
+def generate_holdings(count = 50000, inst = "umich", date = Date.parse("2020-01-01"))
+  # holdings
+  outfile = File.join(OUTDIR, "umich_fake_holdings.ndj")
+  return if warn_exists(outfile)
+  File.open(outfile, "w") do |out|
+    count.times do
+      holding = FactoryBot.build(:holding,
+        organization: "umich",
+        mono_multi_serial: "mono",
+        ocn: rand(MAX_OCN),
+        date_received: date)
+      out.puts(holding.as_document.except("_id").to_json)
+    end
+  end
+end
+
+def htitem_fields(max_ocn: MAX_OCN)
+  htitem = FactoryBot.build(:ht_item, ocns: [rand(max_ocn).to_s])
+  collection = Services.ht_collections[["PU", "MIU", "KEIO", "UCM"].sample]
```

Comment:
Most of this data just gets thrown away. I don't know if it's worth using Faker here, or if it would be better to make it explicit that we don't actually use it.